asset/api/Asset.Samples/AnalyzeIamPolicyLongrunningBigquery.cs

```diff
@@ -57,8 +54,14 @@ public class AnalyzeIamPolicyLongrunningBigquerySample
         var operation = client.AnalyzeIamPolicyLongrunning(request);
         // Wait for it to complete
         operation = operation.PollUntilCompleted();
-        // Return the metadata(request)
-        return operation.Metadata;
+
+        // Check whether the operation failed.
```

Comment:
No need for this - just use operation.Result and it will throw if it's failed.

Reply:
Ah, yes, done.